gymfc/envs/assets/gazebo/plugins/msgs/State.proto

```diff
@@ -24,15 +24,20 @@ message State
   optional float  vbat_voltage = 11;
   optional float  vbat_current = 12;
 
+  // Distance. Units will be in meters as in Gazebo, with two decimals indicating decimeters and centimeters. e.g. 4.85 m (4 meters and 85 centimeters)
```

Comment:
Wrap all lines longer than 80 characters. 
This is too verbose, anyone using this plugin should know what a decimal number is. My previous comment was in regards to you using the "repeated" keyword. If this can have more than one value, does the user need to know anything about how the distance values will be ordered? Perhaps not and the comment should indicate this.

Reply:
Understood